Makefile.inc

```diff
@@ -1,5 +1,4 @@
 , := ,
-_using = $(subst $(,), ,$(using))
```

Comment:
`Makefile.inc` is shared across all projects, this will break any project relying on `_using` (`submariner`, `submariner-charts`, `submariner-operator`).

Reply:
Ok so I guess what I probably need to do is just ensure that in the MAKEFILE we still `include MAKEFILE.inc` so that we still parse the `using=` flag before firing up the clusters, let me make sure this works

Reply:
Fixed